jax/core.py

```diff
@@ -702,7 +702,10 @@ def __repr__(self): return '*'
 unit = Unit()
 literalable_types.add(Unit)
 
-class UnitVar(object):
+class UnitVar(Var):
+  def __init__(self):
```

Comment:
They're read within `Var.__lt__`, which is inherited now that `UnitVar` is a subclass.

Reply:
Gotcha! Can they be class attributes rather than instance attributes, like

```python
class UnitVar(Var):
  count = suffix = None
```

Btw, how does comparing None to an int work? Or is that not what's happening?

Reply:
Comparisons raise a TypeError. My mistake, now fixed. I also switched to class attributes.